contents/blog/posthog-vs-plausible.mdx

```diff
@@ -0,0 +1,178 @@
+---
+title: "In-depth: PostHog vs Plausible"
+date: 2024-10-01
+author:
+ - ian-vanagas
+rootpage: /blog
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/blog/posthog-marketing/marketing-hog.jpg
+featuredImageType: full
+category: General
+tags:
+  - Comparisons
+---
+
+import { ComparisonTable } from 'components/ComparisonTable'
+import { ComparisonHeader } from 'components/ComparisonTable/header'
+import { ComparisonRow } from 'components/ComparisonTable/row'
+
+## How is PostHog different?
+
+### 1. We're free (forever)
+
+Our ["ridiculously cheap" tier](/pricing) includes all of the features of PostHog for free with an allowance of 1M events per month. For Plausible, this would cost a minimum of $69/m. 
+
+Beyond PostHog's free tier, we try to be as cheap as possible and transparent. We can do this because we're efficient. We don't splurge on massive sales teams, or D-list comedians to host an annual convention you'll never attend.
```

Comment:
I don't think this line works in this context because Plausible wouldn't do these things, either.